packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -185,14 +211,47 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
         />
       </Stack>
     ),
-    [itemHovered, menuTitle, ids.participantItemMenuButton]
+    [itemHovered, strings.menuTitle, ids.participantItemMenuButton]
   );
 
   const onDismissMenu = (): void => {
     setItemHovered(false);
     setMenuHidden(true);
   };
 
+  /* @conditional-compile-remove(one-to-n-calling) */
+  /* @conditional-compile-remove(PSTN-calls) */
+  const participantStateString = React.useMemo((): string | undefined => {
+    if (participantState === 'Idle' || participantState === 'Connecting') {
+      return strings?.participantStateConnecting;
+    } else if (participantState === 'EarlyMedia' || participantState === 'Ringing') {
+      return strings?.participantStateRinging;
+    } else if (participantState === 'Hold') {
+      return strings?.participantStateHold;
+    }
+    return;
+  }, [
+    participantState,
+    strings?.participantStateConnecting,
+    strings?.participantStateHold,
+    strings?.participantStateRinging
+  ]);
+
+  const menuItemsWrapperStyle = useMemo(() => {
+    /* @conditional-compile-remove(one-to-n-calling) */
+    /* @conditional-compile-remove(PSTN-calls) */
+    return {
+      /* If participant state is being displayed, hide the menu icon until it is hovered. */
+      display: itemHovered ? 'block' : participantState ? 'none' : 'block'
```

Comment:
I think it's clearer to state this as `display: (!participantState || itemHovered) ? 'block' : 'none'`.

The current conditional is hard to read because it's `'block'` is set in the true and then false nested branch.